### PR TITLE
Revert auto-publish of SOQL extension

### PR DESCRIPTION
**Pull request by praksb**
…(#2794)"

This change reverts auto-publish of SOQL extension

### CODE REVIEW

File: packages/salesforcedx-vscode-soql/package.json [hunk 1]
```diff
@@ -2,9 +2,9 @@
   "name": "salesforcedx-vscode-soql",
   "displayName": "SOQL",
   "description": "Provides code-editing features for SOQL",
-  "qna": "https://github.com/forcedotcom/soql-tooling/issues",
+  "qna": "https://github.com/forcedotcom/salesforcedx-vscode/issues",
```
Comment (lcampos) [inline packages/salesforcedx-vscode-soql/package.json:5]: You might want to still keep this to `soql-tooling` and just remove the script changes below.

Comment (praksb) [reply packages/salesforcedx-vscode-soql/package.json]: fixed

**Comment by lcampos** [comment]
@praksb do we need to port this back to the `develop` branch ? Since we didn't release this week this PR was not automatically ported back to `develop` and next week's release will run using a new branch that gets created out of `develop` on Monday night.

**Comment by praksb** [comment]
we have a pr that @dobladez is working on that we are planning to merge to develop on Monday. This PR would place soql error checks behind a flag. Once that goes through, we would not have a need to port this pr.

**Pull request merged**